public/app/features/provisioning/HomePage.tsx

```diff
@@ -9,11 +9,11 @@ import { Page } from 'app/core/components/Page/Page';
 import { FilesView } from './File/FilesView';
 import GettingStarted from './GettingStarted/GettingStarted';
 import GettingStartedPage from './GettingStarted/GettingStartedPage';
-import { ExportToRepository } from './Repository/ExportToRepository';
-import { MigrateToRepository } from './Repository/MigrateToRepository';
 import { RepositoryActions } from './Repository/RepositoryActions';
 import { RepositoryOverview } from './Repository/RepositoryOverview';
 import { RepositoryResources } from './Repository/RepositoryResources';
+import { ExportToRepository } from './Repository/not-used-yet/ExportToRepository';
```

Comment:
I think you forgot to commit the `not-used-yet` files. 

However, I would simply remove these so that we create them from scratch reusing components and based on a design.

Reply:
🤣  yes -- i came to the same conclusion, but aparntly missed the import paths